website/docs/enterprise/getting-started/install-enterprise-cli.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Install Weave GitOps Enterprise via CLI
+hide_title: true
+toc_max_heading_level: 4
+---
+
+import TierLabel from "../../_components/TierLabel";
+
+# Install Weave GitOps Enterprise via CLI
+
+## Prerequisites
```

Comment:
in terms of experience, we aim for the cli to simplify the install experience, however, given the long list of requirements, the feeling might be somehow different: 

<img width="1297" alt="Screenshot 2023-10-10 at 14 58 21" src="https://github.com/weaveworks/weave-gitops/assets/12957664/dab41ed5-4d14-40fc-aa44-98629a6295ab">

lets keep an eye on list of requirements (to reduce it) and / or may be its just the format and list them as as bullet points helps